data-utils.js

```diff
@@ -383,10 +389,10 @@ export class DataUtils {
         if(child) 
             fullName += ' / ' + child
 
-        let block = await logseq.Editor.appendBlockInPage(page.uuid, `#Metrics ${fullName}`, {
+        let text = this.logseq.settings.journal_title.replaceAll('${metric}', fullName)
+        let block = await logseq.Editor.appendBlockInPage(page.uuid, text, {
             properties: {
-                'metric': fullName,
-                'value': metric.value
+                [fullName]: metric.value
```

Comment:
Property names can not contain spaces (values can). We will need to replace spaces in `fullName` with hyphens, similar to what the Logseq editor does.  I'll merge your PR and commit a change to fix that.